packages-new/components/menu/src/index.ts

```diff
@@ -0,0 +1 @@
+export { OsdsMenu } from './components/osds-menu/public-api';
```

Comment:
actually that`s the only `export *` we`ve kept for now, as filter is already made on public api file
plus you need to export all component here (see select for example)